setup-homebrew/main.sh

```diff
@@ -40,8 +40,14 @@ HOMEBREW_REPOSITORY="$(brew --repo)"
 HOMEBREW_CORE_REPOSITORY="$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-core"
 HOMEBREW_CASK_REPOSITORY="$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-cask"
 
-# Add brew to PATH
-echo "$HOMEBREW_PREFIX/bin" >> $GITHUB_PATH
+# Do in container or on the runner
+if grep -q actions_job /proc/1/cgroup; then
+    # Fix permissions to give normal user access
+    sudo chown -R "$(whoami)" "$HOME" "$PWD/.."
+else
+    # Add brew to PATH
+    echo "$HOMEBREW_PREFIX/bin" >> $GITHUB_PATH
```

Comment:
Is `$HOMEBREW_PREFIX/bin` already in `PATH` when running as a non-root user?

Reply:
Perhaps a comment explaining why this line isn't need when running as a non-root user?

Reply:
It's already in PATH because it it our Docker container.

Reply:
I'm gonna merge this now, feel free to suggest any comments afterwards.